Update 3-add-a-code-example.md (#96)
skills/communicate-using-markdown: .github/steps/3-add-a-code-example.md

_Great job adding an image to the file :tada:_

> In addition, some code blocks should be rendered differently depending on the language; such as JavaScript or command-line text.

Reviewer: ```suggestion
Let's add some code blocks. Code blocks can render differently depending on the language.
```
